README.md

```diff
@@ -139,5 +141,27 @@ var MailExampleApp = React.createClass({
 
 On Android, the `callback` will only be called if an `error` occurs. The `event` argument is unused!
 
+## Example 2
+Check if the user has an email account configured.
+
+```js
+var Mailer = require('NativeModules').RNMail;
+
+Mailer.canSend((error, canSend) => {
```

Comment:
i would prefer to follow rn convention over node. refer: https://github.com/facebook/react-native/blob/4a893ee071f76377a678fe9774190397f636598a/Libraries/CameraRoll/RCTImagePickerManager.m#L49